### PR TITLE
Add metrics page with 'Credentials' Class outline

### LINKED ISSUE #78: Create scripts to build charting metrics; credentials
# 💡 Summary #

Create a a method to pull output data from `db_query.py` for charting. Create a function that manipulates the data to produce an organization's exposed credentials chart.

## Implementation notes ##

- create `pe_reports/metrics.py`
- create a function to handle an organization's exposed credentials
- Import `numpy` package
- Update `setup.py` with new libraries.

## Acceptance criteria ##

- [ ]  run `pip install -e .`
- [ ] run `pre-commit run --all-files`
- [ ] run `pytest /tests`
- [ ] run `pe-reports $(date "+%Y-%m-%d") data/ output/ --db-creds-file=/creds`
- [ ] All code meets dev-standards
- [ ] Changes are limited to a single goal - eschew scope creep!
- [ ] All future TODOs are captured in issues, which are referenced in code comments.
- [ ] All relevant type-of-change labels have been added.

### LINKED ISSUE #96: Update codeowners name schmelz21
# 🐛 Summary #

Github name was change a while back from `schmlez-ctr` to `schmelz21` and needs updated in code owners.

## To update ##
file: .`github/CODEOWNERS`

```
# requested for review when someone opens a pull request.
* @aloftus23 @cduhn17 @dav3r @DJensen94 @elr64 @felddy @jsf9k @mcdonnnj @schmelz-ctr @stewartl97
```

```
# requested for review when someone opens a pull request.
* @aloftus23 @cduhn17 @dav3r @DJensen94 @elr64 @felddy @jsf9k @mcdonnnj @schmelz21 @stewartl97
```

### PR DESCRIPTION
**Pull request by schmelz21**
## 🗣 Description ##

Create a a method to pull modify database data for report output. Create a function that manipulates the data to produce an organization's exposed credentials metrics:

- Total Credentials Exposed
- Credentials with Password
- Distinct Breaches
- Credentials Exposed by Day
- Breach Details

## 💭 Motivation and context ##

These metric scripts introduce the report output metrics taken from the P&E database. 

## 🧪 Testing ##

This current PR needs to pass current `pytest /tests`

## ✅ Checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] _All_ future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by aloftus23 — APPROVED** [review]

**Comment by cduhn17** [comment]
@dav3r @mcdonnnj 
When you get a chance could you review. We have follow up PR's ready to go based on PR#90, but are holding off to get the class approved before we let the next PR out of draft. That way we can make changes to the follow up PR's before official review. Thanks

**Comment by dav3r** [comment]
I will try to review this PR tomorrow or Wednesday.

File: setup.py
Comment (dav3r) [inline setup.py]: Packages here should be sorted alphabetically.

File: src/pe_reports/data/db_query.py
Comment (dav3r) [inline src/pe_reports/data/db_query.py]: Maybe I was on vacation too long last week, but I don't see how `{err}` is supposed to provide different output each of the 4 times it is logged here.

**Comment by dav3r** [comment]
Can you please pull in the latest changes from `develop` before we review this PR?  Many of the changes in this PR already exist in `develop`.  I started reviewing this PR, but quickly got deja vu as I saw a lot of familiar code.

**Comment by cduhn17** [comment]
@dav3r , 

I have the branch related to PR 90 inline with develop and just the changes for this PR are present.

File: src/pe_reports/metrics.py
Comment (dav3r) [inline src/pe_reports/metrics.py]: I'd get rid of this unnecessary `total` variable:
```suggestion
        return df_cred_csg + df_cred_hibp
```

File: src/pe_reports/metrics.py
Comment (dav3r) [inline src/pe_reports/metrics.py]: Please create one or more issues for this TODO and add the issue URL(s) to this comment.

**Review by dav3r — CHANGES_REQUESTED** [review]
I have a few small changes that I'd like to see before I approve.

File: setup.py
Comment (DJensen94) [reply setup.py]: @dav3r I have addressed your recommendation per commit [61c310a](https://github.com/cisagov/pe-reports/pull/90/commits/61c310aee2fe6a6340a64645522144b7722f448d
)

File: src/pe_reports/data/db_query.py
Comment (DJensen94) [reply src/pe_reports/data/db_query.py]: @dav3r I have addressed your recommendation per commit  [538ca59](https://github.com/cisagov/pe-reports/pull/90/commits/538ca592e92d98a5119d282f0e22905ee429c27e)

File: src/pe_reports/metrics.py
Comment (DJensen94) [reply src/pe_reports/metrics.py]: @dav3r I addressed your recommendation per commit [f0fb31e](https://github.com/cisagov/pe-reports/pull/90/commits/f0fb31e5b4050af3f57a8b5bf5228f963de58df8)

File: src/pe_reports/metrics.py
Comment (DJensen94) [reply src/pe_reports/metrics.py]: @dav3r I addressed your recommendation per commit [f0fb31e](https://github.com/cisagov/pe-reports/pull/90/commits/f0fb31e5b4050af3f57a8b5bf5228f963de58df8)

File: src/pe_reports/data/db_query.py
Comment (dav3r) [reply src/pe_reports/data/db_query.py]: Thanks- I thought I might've been missing something funky there.  👍

File: src/pe_reports/metrics.py
Comment (dav3r) [reply src/pe_reports/metrics.py]: Thanks!

**Review by dav3r — APPROVED** [review]
Looks good to me - thanks for responding to my feedback!  🚀

**Pull request merged**
